src/store.ts

```diff
@@ -758,6 +792,21 @@ const createMlStore = (logging: Logging) => {
 
 export const useStore = createMlStore(deployment.logging);
 
+const getDataWindowFromGestures = (gestures: GestureData[]): DataWindow => {
+  const dataLength = gestures.flatMap((g) => g.recordings)[0]?.data.x.length;
+  return dataLength > legacyDataWindow.minSamples
```

Comment:
Should this be >= ?

Reply:
Yes [c35545e](https://github.com/microbit-foundation/ml-trainer/pull/431/commits/c35545ee4b3d7b771344601b6542fe857c018fb0)